src/pilot/config_idrac.py

```diff
@@ -204,6 +204,114 @@ def configure_bios_nics_boot_settings(drac_client, ip_service_tag, pxe_nic_id):
+              non-UEFI, boot protocol is NONE
+    :raises: WSManRequestFailure on request failures
+    :raises: WSManInvalidResponse when receiving invalid response
+    :raises: DRACOperationFailed on error reported back by the iDRAC
+             interface
+    :raises: NotFound when no settings for NIC found
+    """
+    return self.get_nic_legacy_boot_protocol(
+        nic_id).current_value == 'NONE'
+
+
+def is_nic_legacy_boot_protocol_pxe(self, nic_id):
+    """Return true if the legacy, non-UEFI, boot protocol of a NIC is PXE,
+    false otherwise.
+
+    :param nic_id: id of the network interface controller (NIC)
+    :returns: boolean indicating whether or not the legacy,
+              non-UEFI, boot protocol is PXE
+    :raises: WSManRequestFailure on request failures
+    :raises: WSManInvalidResponse when receiving invalid response
+    :raises: DRACOperationFailed on error reported back by the iDRAC
+             interface
+    :raises: NotFound when no settings for NIC found
+    """
+    return self.get_nic_legacy_boot_protocol(nic_id).current_value == 'PXE'
+
+
+def set_nic_legacy_boot_protocol(self, nic_id, value):
+    """Set the legacy, non-UEFI, boot protocol of a NIC.
+
+    If successful, the pending value of the NIC's legacy boot
+    protocol attribute is set. For the new value to be applied, a
+    configuration job must be created and the node must be rebooted.
+
+    :param nic_id: id of the network interface controller (NIC)
+    :param value: legacy boot protocol
+    :returns: dictionary containing a 'commit_required' key with a
+              boolean value indicating whether a configuration job
+              must be created for the new legacy boot protocol
+              setting to be applied
+    :raises: WSManRequestFailure on request failures
+    :raises: WSManInvalidResponse when receiving invalid response
+    :raises: DRACOperationFailed on error reported back by the iDRAC
+             interface
+    """
+    return self.set_nic_setting(nic_id, 'LegacyBootProto', value)
+
+
+def set_nic_legacy_boot_protocol_none(self, nic_id):
+    """Set the legacy, non-UEFI, boot protocol of a NIC to NONE.
+
+    If successful, the pending value of the NIC's legacy boot
+    protocol attribute is set. For the new value to be applied, a
+    configuration job must be created and the node must be rebooted.
+
+    :param nic_id: id of the network interface controller (NIC)
+    :returns: dictionary containing a 'commit_required' key with a
+              boolean value indicating whether a configuration job
+              must be created for the new legacy boot protocol
+              setting to be applied
+    :raises: WSManRequestFailure on request failures
+    :raises: WSManInvalidResponse when receiving invalid response
+    :raises: DRACOperationFailed on error reported back by the iDRAC
+             interface
+    """
+    return self._nic_cfg.set_nic_legacy_boot_protocol(nic_id, 'NONE')
```

Comment:
There is no _nic_cfg variable in config_idrac.  Not sure how this could possibly work.  Has this been tested?